web/l10n/en.json

```diff
@@ -517,5 +517,6 @@
     "categoryVan": "Van",
     "categoryScooter": "Scooter",
     "maintenanceStart": "Start",
-    "maintenancePeriod": "Period"
+    "maintenancePeriod": "Period",
+    "dropzoneText": "Drag and drop a file here or click"
```

Comment:
We use prefixes for string. This one can probably go into "shared" once because it's used in multiple places. So name it something like `sharedDropzoneText` and put next to other shared string.

Reply:
Thx for this review! Solved.